### PR TITLE
Check new, correct type of visibility for contract editor expand/collapse

### PR DESCRIPTION
**Pull request by bcjordan**
Check correct type of visibility for expand/collapse
# Before

badbehavior
# After

goodbehavior

### CODE REVIEW

File: blockly-core/core/ui/contract_editor/contract_editor.js [hunk 1]
```diff
@@ -259,13 +259,15 @@ Blockly.ContractEditor.prototype.setBlockSubsetVisibility = function(isVisible,
 
 Blockly.ContractEditor.prototype.isBlockInFunctionArea = function(block) {
   return block === this.functionDefinitionBlock ||
-    (block.blockSpace === this.modalBlockSpace && block.isUserVisible() &&
+    !this.isAnExampleBlockInEditor_(block) &&
+    (block.blockSpace === this.modalBlockSpace &&
+    block.isVisible() &&
```
Comment (Bjvanminnen) [inline blockly-core/core/ui/contract_editor/contract_editor.js]: Should parens be around all &&'ed statements?

Comment (Bjvanminnen) [inline blockly-core/core/ui/contract_editor/contract_editor.js]: i.e. open paren should be up one line

Comment (Bjvanminnen) [inline blockly-core/core/ui/contract_editor/contract_editor.js]: Actually, can't this just be replaced with isBlockInExampleArea?

Comment (bcjordan) [inline blockly-core/core/ui/contract_editor/contract_editor.js]: Good call, consolidated that logic a bit

**Pull request merged**